actions/promote/triggerNTrack.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable no-await-in-loop */
+/* ************************************************************************
+* ADOBE CONFIDENTIAL
+* ___________________
+*
+* Copyright 2023 Adobe
+* All Rights Reserved.
+*
+* NOTICE: All information contained herein is, and remains
+* the property of Adobe and its suppliers, if any. The intellectual
+* and technical concepts contained herein are proprietary to Adobe
+* and its suppliers and are protected by all applicable intellectual
+* property laws, including trade secret and copyright laws.
+* Dissemination of this information or reproduction of this material
+* is strictly forbidden unless prior written permission is obtained
+* from Adobe.
+************************************************************************* */
+
+const openwhisk = require('openwhisk');
+const { updateExcelTable } = require('../sharepoint');
+const {
+    getAioLogger, PROMOTE_ACTION, PROMOTE_BATCH, actInProgress
+} = require('../utils');
+const appConfig = require('../appConfig');
+const urlInfo = require('../urlInfo');
+const FgStatus = require('../fgStatus');
+const BatchManager = require('../batchManager');
+
+const logger = getAioLogger();
+
+async function main(params) {
+    let payload;
+    appConfig.setAppConfig(params);
+
+    const batchManager = new BatchManager({ key: PROMOTE_ACTION });
+    await batchManager.init();
+    const instanceContent = await batchManager.resumeInstance();
+    // logger.info(`Instance data is ${JSON.stringify(instanceContent)}`);
```

Comment:
please remove this commented line if it's not needed

Reply:
Thanks @arshadparwaiz have removed the commented logs.